pkgs/development/python-modules/nassl/default.nix

```
@@ -0,0 +1,92 @@
+{ stdenv, lib, fetchFromGitHub, fetchurl, buildPythonPackage, zlib, openssl, openssl_1_0_2, pytest, tls-parser }:
+let
+  zlibStatic = zlib.override { static = true; splitStaticOutput = false; };
```

you maybe able to use the static overlay
```suggestion
  zlibStatic = pkgsStatic.zlib
```

---

Yes, thanks! Done.